app/workers/mhv/audit_logout_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module MHV
+  class AuditLogoutJob
+    include Sidekiq::Worker
+
+    sidekiq_options retry: false
+
+    def perform(uuid, mhv_correlation_id)
+      MHVLogging::Client.new(session: { user_id: mhv_correlation_id })
```

Comment:
The problem there is that it would be reasonable for the controller to delete the user. It was doing that for a bit before we changed the behavior. If that happened then we'd never log them out with MHV, since the job is executed after the controller action completes.